src/sidebar/components/annotation-header.js

```diff
@@ -24,6 +24,10 @@ function AnnotationHeaderController(groups, settings, serviceUrl) {
     return persona.isThirdPartyUser(self.annotation.user, settings.authDomain);
   };
 
+  this.thirdPartyUsernameLink = settings.usernameUrl ?
+    settings.usernameUrl + persona.username(this.annotation.user):
```

Comment:
If the annotation's username ever changes, the `thirdPartyUsernameLink` property will fail to update. This is why the data exposed to the template is done via functions which always access the latest version of the annotation.